src/model/room/Room.java

```diff
@@ -0,0 +1,67 @@
+package model.room;
+
+public abstract class Room {
+    protected String status = "Available";
+    protected int noOfRoom;
```

Comment:
[Improvement] o sugestie, schimba in roomNumber, sau roomNo, ca sa fie mai clar, fiindca vad ca in Suite pasezi si noOfRoom si noOfRooms si un programator in graba ar putea sa le confunde si sa le puna aiurea